docs/data-sources/logdna_alert.md

```diff
@@ -0,0 +1,66 @@
+# Data Source: `logdna_alert`
+
+Pulls in the relevant details from existing [LogDNA Preset Alerts](https://docs.logdna.com/docs/alerts). The `logdna_alert` _data source_ remotely fetches the information from a preset alert using its ID. This preset alert may or may not be directly managed by Terraform. For the management of preset alerts as Terraform _resources_, refer to the documentation [here](../resources/logdna_alert.md).
+
+Users may opt to reference certain preset alerts as data sources rather than definining them as resources. In this scenario, the preset alerts would not be managed as Terraform _resources_ but the data can still be accessed and referenced in other modules.
+
+To create a `logdna_alert` data source, the `presetid` argument must be provided in its declaration to ensure the data being read remotely is correct.
+
+## Example Usage
+
+```hcl
+provider "logdna" {
+  servicekey = "xxxxxxxxxxxxxxxxxxxxxxxx"
+  url = "https://api.logdna.com" # (Optional) specify a LogDNA region
+}
+
+resource "logdna_alert" "my_alert" {
+  name = "My Preset Alert via Terraform"
+  email_channel {
+    emails          = ["test@logdna.com"]
+    immediate       = "false"
+    operator        = "presence"
+    triggerlimit    = 15
+    triggerinterval = "15m"
+    terminal        = "true"
+    timezone        = "Pacific/Samoa"
+  }
+}
+
+# create data source by referencing the ID from an alert declared in the same config
+data "logdna_alert" "my_alert_remote" {
```

Comment:
Super minor nit: should we call this `my_alert_local` as the resource is declared in the same file and the other one `my_alert_remote` (instead of `external_remote`)?

Reply:
The change looks good 👍